src/main/java/Note/ui/Note.java

```diff
@@ -0,0 +1,146 @@
+package Note.ui;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * The Note class represents a simple command-line task manager chatbot.
+ * Delegates responsibilities to Ui, TaskList, Parser, and Storage.
+ */
+public class Note {
+    private static final String FILE_PATH = "data/duke.txt";
+
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    public Note() {
+        ui = new Ui();
+        storage = new Storage(FILE_PATH);
+        try {
+            tasks = new TaskList(storage.load());
+        } catch (IOException e) {
+            ui.showLoadingError();
+            tasks = new TaskList();
+        }
+    }
+
+    public void run() {
+        ui.showWelcome();
+
+        while (true) {
+            String input = ui.readCommand();
+            String command = Parser.getCommand(input);
+            String arguments = Parser.getArguments(input);
+
+            try {
+                switch (command) {
+                    case "bye":
```

Comment:
I think it would be better to align the switch and case statements to better match the coding guidelines.
```suggestion
                switch (command) {
                case "bye":
```

Reply:
Seconded!